### DISCUSSION TITLE
Featuring a component on Dioxus community.

### DISCUSSION THREAD

**Discussion by vnermolaev** [DioxusLabs/dioxus#2646] [Q&A] [answered]
Hey there! @matthunz @marc2332 @nissy-dev

I’m working on a chessboard component with Dioxus and would love for you to check it out! You can find it [here](https://github.com/vnermolaev/dioxus-chessboard).

I’d be absolutely delighted to get my component featured on the [Dioxus Community](https://github.com/dioxus-community) website. Is there a process for submitting or getting selected for features? Any guidance would be greatly appreciated!

Thanks so much! 😊

**Comment by marc2332** [comment, marked as answer]
Hey @vnermolaev, the Dioxus Community is for others to work together and support older crates used by dioxus users. Perhaps you wanted it to be showcased in https://dioxuslabs.com/awesome instead?

**Reply by vnermolaev** [reply]
Yes, indeed! Thanks @marc2332
